File: mymedia-services/src/main/java/mymediaMain/model/PersonalData.java
package mymediaMain.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.io.Serializable;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class PersonalData implements Serializable {
    private String username;
    private String email;
    private String phoneNumber;
    private String bio;
    private String fullName;
}

File: mymedia-frontend/src/app/profile/Model/PersonalData.ts
import {MymediaResponse} from '../../shared/model/Shared';

export interface PersonalData {
  bio: string;
  email: string;
  fullName: string;
  phoneNumber: string;
  username: string;
}

export interface PersonalResponse extends MymediaResponse {
  personalData: PersonalData;
}

export interface FollowersAndFollowingResponse extends MymediaResponse {
  following: number;
  followers: number;
  posts: number;

}
